dnsimple/dnsimple-support
# Dark theme

This implements a dark theme akin to the developer site. The background is a bit different than the developer site on purpose so it's easy to differentiate them if you're browsing between the 2.

## QA

Use the Netlify preview to check:
- Alerts
- `pre` and `code` blocks
- H1, H2, H3 titles
- Switching between themes. It should default to your OS theme when browsing for the first time.
